src/pages/debrief/debrief.ts

```diff
@@ -28,6 +48,49 @@ export class DebriefPage extends BasePageComponent {
 
   }
 
+  ngOnInit(): void {
+    this.pageState = {
+      seriousFaults$:  this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
+        select(getTestData),
+        map(data => getSeriousOrDangerousFaults(data.seriousFaults)),
+      ),
+      dangerousFaults$: this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
+        select(getTestData),
+        map(data => getSeriousOrDangerousFaults(data.dangerousFaults)),
+      ),
+      drivingFaults$: this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
+        select(getTestData),
+        map(data => getDrivingFaults(data.drivingFaults)),
+      ),
+      drivingFaultCount$: this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
+        select(getTestData),
+        map((data) =>  {
+          const faults = getDrivingFaults(data.drivingFaults);
+          return faults.reduce((sum, c) => sum + c.count, 0);
+        }),
+      ),
+    };
+
+    const { seriousFaults$, dangerousFaults$, drivingFaults$ } = this.pageState;
+
+    const merged$ = merge(
+      seriousFaults$,
+      dangerousFaults$,
+      drivingFaults$,
+    );
+
+    this.subscription = merged$.subscribe();
```

Comment:
Need to unsubscribe from this at some point.

Reply:
added an unsubscribe.